Gems/AtomTressFX/Assets/Shaders/HairLightTypes.azsli

```diff
@@ -0,0 +1,633 @@
+    inout Surface surface, 
+    inout LightingData lightingData)
+{
+    // Direct towards close to half vector by never cross fully (more weight to camera direction)
+    float3 biNormal;
+    if (o_hairLightingModel == HairLightingModel::GGX)
+    {
+        float3 halfDir = normalize( dirToLight + 1.2 * lightingData.dirToCamera);
+        biNormal = normalize(cross(tangent, halfDir));
+    }
+    else
+    {
+        biNormal = normalize(cross(tangent, lightingData.dirToCamera));   // Original - simple face forward
+    }
+
+    float3 projectedNormal = cross(biNormal, tangent);
+    surface.normal = normalize(projectedNormal);    // the normalization might be redundunt 
+
+    // Next is important in order to set NdotV and other PBR settings - needs ot be set once per light
+    UpdateLightingParameters(lightingData, surface.position, surface.normal, surface.roughnessLinear);
+
+    // Diffuse and Specular response
+    lightingData.specularResponse = FresnelSchlickWithRoughness(lightingData.NdotV, surface.specularF0, surface.roughnessLinear);
+    lightingData.diffuseResponse = 1.0f - lightingData.specularResponse;
+}
+
+//==============================================================================
+#include <Atom/Features/PBR/Lights/LightTypesCommon.azsli>
+//==============================================================================
+
+
+//==============================================================================
+//                         Simple Point Light 
+//==============================================================================
+void ApplySimplePointLight(ViewSrg::SimplePointLight light, Surface surface, inout LightingData lightingData)
+{  
+    float3 posToLight = light.m_position - surface.position;
+    float d2 = dot(posToLight, posToLight); // light distance squared
+    float falloff = d2 * light.m_invAttenuationRadiusSquared;
+
+    // Only calculate shading if light is in range
+    if (falloff < 1.0f)
+    {
+        // Smoothly adjusts the light intensity so it reaches 0 at light.m_attenuationRadius distance
+        float radiusAttenuation = 1.0 - (falloff * falloff);
+        radiusAttenuation = radiusAttenuation * radiusAttenuation;
+
+        // Standard quadratic falloff
+        d2 = max(0.001 * 0.001, d2); // clamp the light to at least 1mm away to avoid extreme values.
+        float3 lightIntensity = (light.m_rgbIntensityCandelas / d2) * radiusAttenuation;
+
+        float3 dirToLight = normalize(posToLight);
+        SetNormal(surface.tangent, dirToLight, surface, lightingData);
+
+        // Diffuse contribution
+        lightingData.diffuseLighting += GetDiffuseLighting(surface, lightingData, lightIntensity, dirToLight);
+
+        // Specular contribution
+        lightingData.specularLighting += GetSpecularLighting(surface, lightingData, lightIntensity, dirToLight);
+    }
+}
+
+void ApplyCulledSimplePointLights(Surface surface, inout LightingData lightingData)
```

Comment:
Is this used anywhere? If not, can you add a comment saying it's temporary/what it's future use will be? Thanks
Maybe also comment out the function so it's obvious it's not used currently.

Reply:
done - tnx for noticing.